bumble/gatt_client.py

```diff
@@ -558,23 +555,31 @@ async def subscribe(self, characteristic, subscriber=None):
             logger.warning('subscribing to characteristic with no CCCD descriptor')
             return
 
-        # Set the subscription bits and select the subscriber set
-        bits = ClientCharacteristicConfigurationBits.DEFAULT
-        subscriber_sets = []
-        if characteristic.properties & Characteristic.NOTIFY:
-            bits |= ClientCharacteristicConfigurationBits.NOTIFICATION
-            subscriber_sets.append(self.notification_subscribers.setdefault(characteristic.handle, set()))
-        if characteristic.properties & Characteristic.INDICATE:
-            bits |= ClientCharacteristicConfigurationBits.INDICATION
-            subscriber_sets.append(self.indication_subscribers.setdefault(characteristic.handle, set()))
+        def characteristic_is_notify(characteristic, prefer_notify):
+            if (
```

Comment:
suggest: more readable as: `if characteristic.properties & Characteristic.NOTIFY and characteristic.properties & Characteristic.INDICATE`

Reply:
parens aren't needed

Reply:
This is the formatting that black chose, so I'll stick with it.